test/coreneuron/test_direct.hoc

```diff
@@ -61,9 +62,20 @@ proc test_direct_memory_transfer() { localobj po, pc, ic, tv, vvec, i_mem, tvstd
     pc.psolve(tstop)
 
     // compare results
-    print (tv.eq(tvstd))
-    print (vvec.cl().sub(vstd).abs().max() < 1e-10)
-    print (i_mem.cl().sub(i_memstd).abs().max() < 1e-10)
+    result1 = tv.eq(tvstd)
+    result2 = vvec.cl().sub(vstd).abs().max() < 1e-10
```

Comment:
See comment in `test/coreneuron/test_units.py` about tolerance

Reply:
I've left this one the same for now. Note that this PR is not changing any thresholds here, and I would rather not get distracted just now by surveying which compilers/optimisation settings need which tolerances.

Reply:
yes, this we can keep as it is.